index.html

```diff
@@ -9,7 +9,7 @@
 </head>
 <body>
   <header>
-    <h1 id ="hello">Hello Microverse!,</h1>
+    <h1 id ="hello">Hello Microverse!</h1>
   </header>
   <main></main>
   <footer></footer>
```

Comment:
- 👏 Great job !!!
- [ ] Writing empty tags or adding extra lines to code makes it kinda messy, and this is why by convention, we should try as much as we can to avoid such in our code base. While these are valid tags, you do not have any content for them at the moment. How about removing them to clean up our code? 😉